src/test/java/space/obminyashka/items_exchange/service/UserServiceIntegrationTest.java

```diff
@@ -105,14 +104,6 @@ void testPermanentlyDeleteUsers_ShouldDeleteRequiredUsers() {
         }
     }
 
-    @Test
-    void makeAccountActiveAgain_WhenDataCorrect_Successfully() {
```

Comment:
All the test cases have to be created according to the [test pyramid](https://martinfowler.com/articles/practical-test-pyramid.html).
That means that the same logic has to be covered with the unit, integration, end 2 end tests.
You shouldn't remove the test when you have an end 2 end test for the same part of the code but update that integration test as well.
Each test has its own purpose: end 2 end - check the whole flow; integration - work with Spring context and beans; unit - no Spring context and describes only single behavior of a tested method.

Reply:
Could you please update it here rather than move it up?